tests/unit/location.test.ts

```
@@ -0,0 +1,122 @@
+
+    it("should format distances greater than 1km in kilometers", () => {
+      expect(formatDistance(1)).toBe("1.0km");
+      expect(formatDistance(1.5)).toBe("1.5km");
+      expect(formatDistance(10.25)).toBe("10.3km");
+    });
+  });
+});
+
+// Test API response structure
+describe("Location API", () => {
+  it("should return correct response structure", async () => {
```

ESLint

[error] Async arrow function has no 'await' expression.

(require-await)